src/bptl/camunda/utils.py

```diff
@@ -149,3 +155,26 @@ def fail_task(task: ExternalTask, reason: str = "") -> None:
     }
 
     camunda.post(f"external-task/{task.task_id}/failure", json=body)
+
+
+def log_camunda_error(task: ExternalTask, exc: requests.HTTPError) -> None:
+    # log Camunda errors if we get any at all
+    response = getattr(exc, "response", None)
+    if response is None:
+        return
+
+    try:
+        error_information = response.json()
+    except json.JSONDecodeError:
+        error_information = None
+
+    TimelineLog.objects.create(
+        content_object=task,
+        template="timeline_logger/camunda/failed_complete.txt",
```

Comment:
I can't find this template file in the project

Reply:
it doesn't exist :-) I'll add something